index.html

```diff
@@ -57,64 +57,34 @@ <h1>
         Last Modified:
       </dt>
       <dd>
-        28 June 2019
+        1 January 2020
       </dd>
     </dl>
     <h2 id="goals">
       Goals
     </h2>
     <p>
-      The <a href="https://www.w3.org/community/webscreens/">Second Screen
-      Community Group (CG)</a> explores the use of secondary screens from Web
-      pages. This Community Group incubates and develops specifications of
-      network protocols that implement the <a href=
-      "https://w3c.github.io/presentation-api/">Presentation API</a> and the
-      <a href="https://w3c.github.io/remote-playback/">Remote Playback API</a>.
-      In addition, this Community Group incubates new additions to the APIs
-      defined in the Second Screen WG.
+      The mission of the <a href=
+      "https://www.w3.org/community/webscreens/">Second Screen Community Group
+      (CG)</a> is to explore, incubate, and define interfaces that enable new
+      multi-display and multi-window computing user experiences on the Web.
+      This Community Group defines new additions to the <a href=
+      "https://w3c.github.io/presentation-api/">Presentation API</a> and
```

Comment:
The WG charter in review says that new features for the Remote Playback API will be incubated here, so I think the Remote Playback API should also be listed.

Reply:
Correct, added the Remote Playback API here.